lib/config/accountsConfig.js

```diff
@@ -0,0 +1,78 @@
+var mySubmitFunc = function(error, state){
+  if (!error) {
```

Comment:
Let's remove all stubs/examples from committed code. You can link to docs at the top of a file (url) if you'd like, or you can just omit them and whoever is implementing it will look it up.